illumina.py

```diff
@@ -105,6 +105,29 @@ def main_illumina_demux(args):
     else:
         samples = illumina.get_SampleSheet(only_lane=args.lane)
 
+
+    link_locs="false"
+    # For HiSeq-4000/X runs, If Picard's CheckIlluminaDirectory is
+    # called with LINK_LOCS=true, symlinks with absolute paths
+    # may be created, pointing from tile-specific *.locs to the 
+    # single s.locs file in the Intensities directory.
+    # These links may break if the run directory is moved.
+    # We should begin by removing broken links, if present,
+    # and call CheckIlluminaDirectory ourselves if a 's.locs'
+    # file is present
+    if os.path.exists(os.path.join(illumina.get_intensities_dir(), "s.locs")):
+        # recurse to remove broken links in directory
```

Comment:
Perhaps add a `log.info` message here just so we can trace after the fact whether this is happening or not?